lib/database/user.php

```diff
@@ -265,7 +265,8 @@ function getUserPageInfo(&$user, &$libraryOut, $numGames, $numRecentAchievements
     }
     $libraryOut['ContribCount'] = $userInfo['ContribCount'];
     $libraryOut['ContribYield'] = $userInfo['ContribYield'];
-    $libraryOut['TotalPoints'] = $userInfo['RAPoints'];
+    $libraryOut['TotalHardcorePoints'] = $userInfo['RAPoints'];
+    $libraryOut['TotalSoftcorePoints'] = $userInfo['RASoftcorePoints'];
```

Comment:
This change will impact the public API:
https://github.com/RetroAchievements/RAWeb/blob/f366b690c3a06565b6b1d1900c644edc0d6b2e0d/public/API/API_GetUserSummary.php#L9-L11

Recommend duplicating `TotalHardcorePoints` into `TotalPoints` there, as is already being done for `Points`:
https://github.com/RetroAchievements/RAWeb/blob/f366b690c3a06565b6b1d1900c644edc0d6b2e0d/public/API/API_GetUserSummary.php#L106

Reply:
I believe I've taken care of this and the other API call now but I'll let you verify that I did it correctly as I'm still not 100% sure.

Reply:
Looks good.

API_GetUserSummary:
```
   "TotalPoints":"2900",
   "TotalSoftcorePoints":"25",
   "TotalTruePoints":"4268",
...
   "Points":"2900",
   "SoftcorePoints":"25",
```
API_GetUserRankAndScore:
```
    "Score":2900,
    "SoftcoreScore":25
```
User page:
![image](https://user-images.githubusercontent.com/32680403/175044900-fba14738-090a-4383-9318-c3632d560dd4.png)